src/guide/introduction.md

```diff
@@ -87,138 +87,138 @@ Vue.createApp(AttributeBindingApp).mount('#bind-attribute')
 
```

Comment:
From here down, it's just that I swapped two sections around, but it's similar enough that weird diffing is happening